vendor/github.com/cilium/ebpf/abi.go

```diff
@@ -35,7 +34,7 @@ func newMapABIFromSpec(spec *MapSpec) *MapABI {
 func newMapABIFromFd(fd *internal.FD) (string, *MapABI, error) {
 	info, err := bpfGetMapInfoByFD(fd)
 	if err != nil {
-		if xerrors.Is(err, syscall.EINVAL) {
+		if errors.Is(err, syscall.EINVAL) {
```

Comment:
I like this change overall, but I wonder if we should actually make these changes to vendored dependencies? Can there be any problems when we upgrade dependencies?

Reply:
This was done through cilium/ebpf#116 which is vendored in by commit a1cd0b899a7f135d570402375e14104a3505a461 in this PR.

Reply:
The ebpf dep has been bumped and had similar work (see https://github.com/cilium/ebpf/pull/116).